packages/runtime/container-runtime/src/channelCollection.ts

```diff
@@ -812,11 +812,11 @@ export class ChannelCollection implements IFluidDataStoreChannel, IDisposable {
 		// loaded, as this client is playing the role of creating client,
 		// and creating clients always create realized data store contexts.
 		const channel = await dataStoreContext.realize();
-		await channel.entryPoint.get();
-
 		// add to the list of bound or remoted, as this context must be bound
 		// to had an attach message sent, and is the non-detached case is remoted.
 		this.contexts.addBoundOrRemoted(dataStoreContext);
```

Comment:
would it make sense to do the binding before the realization?

Reply:
The idea being that the context should be fully setup before we realize, and get the handle, rather than happening between realization, and getting the handle?

Reply:
Yeah that makes more sense. Let me try that